lib/rabbit/logger.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Rabbit
+  class Logger
+    attr_reader :loggers
+
+    def self.wrap(*loggers)
```

Comment:
Why do we add multiple loggers functionality and then don't use it?

Reply:
Because some projects needs several loggers for one action, for example, write to log file and send to the InfluxDB with custom logger

Reply:
If you don't need it, everything will still work out of the box, but now you can change the default values

Reply:
OK, it just looks a bit strange that we add such non-related and generic functionality to `rabbit_messaging` gem...